worldedit-core/src/main/java/com/sk89q/worldedit/util/io/ResourceLoader.java

```diff
@@ -40,4 +40,17 @@ public static URL getResource(Class clazz, String name) throws IOException {
         }
         return url;
     }
+
+    public static URL getResourceRoot(String name) throws IOException {
```

Comment:
Can you make the other method delegate to this one, or vice versa if you think that's easier? Would like to not repeat the modjar hack.